tools/scripts/rsyslog-4xx-recovery

```diff
@@ -31,23 +29,6 @@ def main():
         except ValueError as e:
             write_stderr(str(e))
 
-        # now decide what do to based on eventnames
-        if headers["eventname"] == "PROCESS_STATE_FATAL":
-            headers.update(
-                dict(
-                    [x.split(":") for x in sys.stdin.read(int(headers["len"])).split()]
```

Comment:
For any historians that come back to this PR comment.

We aren't sure that this ever worked. Or, if it did, it was a time when the header and data were not read in the general while loop above.

This `sys.stdin.read()` would consume outside of the current message boundaries. We think maybe reading another message from supervisor. The logic that follow only "replies" once. This could/would lead to the supervisor buffer backing up.